lib/widgets/compose_box.dart

```diff
@@ -1343,6 +1344,20 @@ class _SendButtonState extends State<_SendButton> {
         message: message);
       return;
     }
+
+    store = PerAccountStoreWidget.of(context);
+    final destination = widget.getDestination();
```

Comment:
The point is this is the destination we just sent to, right? Perhaps save that earlier in a variable, vs. recomputing here, to make that clearer.